content/hook-set-codes.json

```diff
@@ -0,0 +1,434 @@
+[
+  {
+    "code": 1,
+    "identifier": "AMENDMENT_DISABLED",
+    "description": "attempt to HookSet when amendment is not yet enabled."
+  },
+  {
+    "code": 2,
+    "identifier": "API_ILLEGAL",
+    "description": "HookSet object contained HookApiVersion for existing HookDefinition"
+  },
+  {
+    "code": 3,
+    "identifier": "API_INVALID",
+    "description": "HookSet object contained HookApiVersion for unrecognised hook API "
+  },
+  {
+    "code": 4,
+    "identifier": "API_MISSING",
+    "description": "HookSet object lacked HookApiVersion"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
+  {
+    "code": 5,
+    "identifier": "BLOCK_ILLEGAL",
+    "description": " a block end instruction moves execution below depth 0 {{}}`}` <= like this"
+  },
```

Comment:
hmm - why is the above repeated 6 times?

Reply:
Goddam copilot 😅